{"name": "test_01", "status": "failed", "statusDetails": {"message": "AssertionError: assert 1 == 100", "trace": "self = <test_02login.TestLogin object at 0x7fd52d8729a0>\n\n    def test_01(self):\n>       assert 1 == 100\nE       assert 1 == 100\n\ntestcase/test_02login.py:53: AssertionError"}, "start": 1612402169817, "stop": 1612402169818, "uuid": "723895d8-bc56-471d-91d5-9520ac14da5b", "historyId": "eacdf5b357d0df8e82a22a5fa83b1eac", "testCaseId": "a7d0e662833bee03a556a4c05bec2ebf", "fullName": "testcase.test_02login.TestLogin#test_01", "labels": [{"name": "tag", "value": "han"}, {"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_02login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "bogon"}, {"name": "thread", "value": "31485-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_02login"}]}